[Content] Black Belt Story

acblackford opened the pull request:
<!-- -----------^ Click "Preview" for a functional view! -->

## Why are you creating this Pull Request?

- [Adding Datasets or Stories](?title=Content%3A%20%3Cname%3E&expand=1&template=content.md)
- [Version Release](?title=Deploy%20vX.X.X&expand=1&template=version_release.md)
- [Other](?expand=1&template=default.md)

acblackford committed:
fixed block with interactive soil map

acblackford commented:
@anayeaye @slesaad the soil texture dataset in this PR is ready for prod

acblackford committed:
removed display flex calls from blocks

acblackford committed:
moved svi stuff to scrollytelling

acblackford committed:
fixed some scrollytelling centers, counties still to add

acblackford committed:
fixed scrollytelling map zoom and centers

acblackford committed:
changed county cards in scrollytelling to socioeconomic svi data

acblackford committed:
added updated age trend plot

acblackford committed:
added caption to new figure

acblackford committed:
fixing indents on new image call

anayeaye commented on stories/black-belt-climate-ej.stories.mdx:
```diff
@@ -296,10 +266,9 @@ taxonomy:
     <p>
-     **Editor**: Maheshwari Neelam and Derek Koehl; 
+     **Editors**: Maheshwari Neelam and Derek Koehl; 
      **Developers**: Andrew Blackford, Trent Cowan, Jerika Christman, Brian Freitag, and Aaron Kaulfus ;
```
```suggestion
     **Developers**: Andrew Blackford, Trent Cowan, Jerika Christman, Brian Freitag, and Aaron Kaulfus;
```

anayeaye reviewed (commented):
remove space and try trigger deploy preview

anayeaye committed:
Update stories/black-belt-climate-ej.stories.mdx

hanbyul-here committed:
Adjust image size

slesaad commented:
this image is looking too small now
![image](https://github.com/user-attachments/assets/e788f2e0-337f-41cf-aa20-05c385ad55d4)

also, for the line charts used, using the `<Chart>` component will be better than using images

slesaad commented on stories/black-belt-climate-ej.stories.mdx:
```diff
@@ -296,10 +266,9 @@ taxonomy:
     <p>
-     **Editor**: Maheshwari Neelam and Derek Koehl; 
-     **Developers**: Andrew Blackford, Trent Cowan, Jerika Christman, Brian Freitag, and Aaron Kaulfus ;
+     **Editors**: Maheshwari Neelam and Derek Koehl; 
+     **Developers**: Andrew Blackford, Trent Cowan, Jerika Christman, Brian Freitag, and Aaron Kaulfus;
      **Science and Content Contributors**: Maheshwari Neelam and Udaysankar Nair;
```
```suggestion
     **Editors**: Maheshwari Neelam and Derek Koehl;
     
     **Developers**: Andrew Blackford, Trent Cowan, Jerika Christman, Brian Freitag, and Aaron Kaulfus;
     
     **Science and Content Contributors**: Maheshwari Neelam and Udaysankar Nair;
```

if you want them to be on separate lines (which I think will look better), do this ☝️

slesaad commented on stories/black-belt-climate-ej.stories.mdx:
```diff
@@ -259,18 +228,19 @@ taxonomy:
-  </Prose>
+<Block style={{justifyContent: 'center', alignItems: 'center', margin: '0 auto', width: '80%' }}>
   <Figure style={{ flex: '0 0 auto'}}>
     <Image
       src={new URL('./BB_Estab_AL.png', import.meta.url).href}
       style={{ maxWidth: '800px', height: 'auto' }}
     />
```
these images are blurry, maybe use the [`<Chart>` component](https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/content/MDX_BLOCKS.md#chart)?

acblackford committed:
trying to fix image size in scrollytelling

mahineelam committed:
added improved figures, made content changes, and addressed comments

mahineelam committed:
right images

mahineelam committed:
text edits

mahineelam committed:
error fixed

mahineelam committed:
changes

mahineelam committed:
CHANGES

mahineelam committed:
svi_changes

mahineelam committed:
rearrange text

mahineelam committed:
fix image

mahineelam committed:
fix image

mahineelam committed:
fix block

mahineelam committed:
image width

mahineelam committed:
minor fixes

mahineelam committed:
error fixed

mahineelam committed:
units and text

acblackford committed:
fixed DOI in paper

mahineelam reviewed (commented):
edits to text

mahineelam committed:
text changes

jerikac commented:
Hey Code Owners - this PR is ready for review!

aboydnw reviewed (approved):
@acblackford merge when ready

@hanbyul-here @snmln let's include this in the release today if possible

aboydnw commented:
Actually, nevermind @hanbyul-here @snmln I think this story is still dependent on this ticket: https://github.com/NASA-IMPACT/veda-backend/issues/441

mahineelam committed:
correction

jerikac commented:
Now that [#441 ](https://github.com/NASA-IMPACT/veda-backend/issues/441) is closed, can we push this story to Prod? @aboydnw

aboydnw commented:
@mahineelam looks like you've been making the bulk of the changes. when you're ready go ahead and merge.

I just did another quick smoke test and looks good. If you merge here then we can plan to release to production the week of Nov 18th

aboydnw commented:
@mahineelam based on that proposed release to production then you might want to change the published date before merging

jerikac committed:
Update publication date  to Nov 11

Original Pub date was old -- updated to sync with next VEDA release (Nov11 or Nov12, 2024)

dzole0311 committed:
Merge branch 'develop' into black-belt
